api/mapping/models.py

```diff
@@ -57,6 +57,46 @@ class ScanReportValue(BaseModel):
     value = models.CharField(max_length=32)
     frequency = models.IntegerField()
 
+# class UsagiProperties(BaseModel):
```

Comment:
Added a skeleton model structure for Usagi properties and to store Usagi results. This needs a lot of work, but it's a start.